main.go

```diff
@@ -215,20 +216,23 @@ func RollingBackupAction(c *cli.Context) error {
 		if len(backupName) == 0 {
 			backupName = fmt.Sprintf("%s_etcd", time.Now().Format(time.RFC3339))
 		}
-		return CreateBackup(backupName, etcdCACert, etcdCert, etcdKey, etcdEndpoints, client, s3Config)
+		return CreateBackup(backupName, etcdCACert, etcdCert, etcdKey, etcdEndpoints, client, bc)
 	}
 	backupTicker := time.NewTicker(creationPeriod)
 	for {
 		select {
 		case backupTime := <-backupTicker.C:
 			backupName := fmt.Sprintf("%s_etcd", backupTime.Format(time.RFC3339))
-			CreateBackup(backupName, etcdCACert, etcdCert, etcdKey, etcdEndpoints, client, s3Config)
+			CreateBackup(backupName, etcdCACert, etcdCert, etcdKey, etcdEndpoints, client, bc)
```

Comment:
what happens if this call of the DeleteBackups() calls fail ? we don't report errors ?

Reply:
It will log the error message but won’t report back to any other services because it is recurring actions, also it will retry in every interval hours.